### PR TITLE
Add v3 transactions additional fields

### PR DESCRIPTION
**Pull request by ArielElp**
This change is [Reviewable](https://reviewable.io/reviews/starkware-libs/starknet-specs/162)

### CODE REVIEW

File: api/starknet_api_openrpc.json [hunk 1]
```diff
@@ -1882,6 +1882,42 @@
                                 "title": "L1 Gas",
                                 "description": "The max amount and max price per unit of L1 gas used in this tx",
                                 "$ref": "#/components/schemas/RESOURCE_LIMITS"
+                            },
+                            "l2_gas": {
+                                "title": "L2 Gas",
+                                "description": "The max amount and max price per unit of L2 gas used in this tx",
+                                "$ref": "#/components/schemas/RESOURCE_LIMITS"
+                            },
```
Comment (Yael-Starkware) [inline api/starknet_api_openrpc.json:1890]: Note the structure of l1_gas, l2_gas is different in the GW and FGW, would be nicer if they'd be the same:
    "resource_bounds": {
        "L1_GAS": {
            "max_amount": "0x5",
            "max_price_per_unit": "0x6"
        },
        "L2_GAS": {
            "max_amount": "0x0",
            "max_price_per_unit": "0x0"
        }
    },

File: api/starknet_api_openrpc.json [hunk 2]
```diff
@@ -1892,7 +1928,13 @@
                             "signature",
                             "nonce",
                             "class_hash",
-                            "l1_gas"
+                            "l1_gas",
+                            "l2_gas",
+                            "tip",
+                            "paymaster_data",
+                            "account_deployment_data",
```
Comment (Yael-Starkware) [inline api/starknet_api_openrpc.json:1935]: why is "account_deployment_data" relevant for declare?

**Review by ArielElp — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/starkware-libs/starknet-specs/162)* status: 0 of 1 files reviewed, 2 unresolved discussions (waiting on @Yael-Starkware)

___
*[`api/starknet_api_openrpc.json` line 1890 at r1](https://reviewable.io/reviews/starkware-libs/starknet-specs/162#gh-1387584780:-Ninl8juCIaECYkNhcKF:b-896fix) ([raw file](https://github.com/starkware-libs/starknet-specs/blob/79b3a5d4639623e136ff3e2ccf5f0c53bd68353c/api/starknet_api_openrpc.json#L1890)):*
<details><summary><i>Previously, Yael-Starkware (yael-starkware) wrote…</i></summary><blockquote>

Note the structure of l1_gas, l2_gas is different in the GW and FGW, would be nicer if they'd be the same:
    "resource_bounds": {
        "L1_GAS": {
            "max_amount": "0x5",
            "max_price_per_unit": "0x6"
        },
        "L2_GAS": {
            "max_amount": "0x0",
            "max_price_per_unit": "0x0"
        }
    },

</blockquote></details>

Done.

**Review by ArielElp — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/starkware-libs/starknet-specs/162)* status: 0 of 1 files reviewed, 2 unresolved discussions (waiting on @Yael-Starkware)

___
*[`api/starknet_api_openrpc.json` line 1935 at r1](https://reviewable.io/reviews/starkware-libs/starknet-specs/162#gh-1387591929:-NioPTwR37Z-w-zUvz-K:bo1hdsk) ([raw file](https://github.com/starkware-libs/starknet-specs/blob/79b3a5d4639623e136ff3e2ccf5f0c53bd68353c/api/starknet_api_openrpc.json#L1935)):*
<details><summary><i>Previously, Yael-Starkware (yael-starkware) wrote…</i></summary><blockquote>

why is "account_deployment_data" relevant for declare?
</blockquote></details>

still not clear if we'll keep it, will be decided in the next RC

**Review by Yael-Starkware — APPROVED** [review]
> Reviewed 1 of 1 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/starkware-libs/starknet-specs/162)* status: all files reviewed, 2 unresolved discussions (waiting on @ArielElp)

**Review by Yael-Starkware — APPROVED** [review]
:lgtm:

> *[Reviewable](https://reviewable.io/reviews/starkware-libs/starknet-specs/162#-:-NiokmlX2JsGEvQUV2zv:bnfp4nl)* status: all files reviewed, 2 unresolved discussions (waiting on @ArielElp)

**Review by ArielElp — COMMENTED** [review]
> Dismissed @Yael-Starkware from 2 discussions.
> *[Reviewable](https://reviewable.io/reviews/starkware-libs/starknet-specs/162)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @ArielElp)

**Pull request merged**